scriptworker/constants.py

```diff
@@ -206,6 +206,7 @@
                 "netlocs": ("github.com", ),
                 "path_regexes": (
                     r"^(?P<path>/mozilla-mobile/focus-android)(/|.git|$)",
+                    r"^(?P<path>/mozilla-mobile/android-components)(/|.git|$)",
```

Comment:
Should we add `reference-browser` in advance for the work that's upcoming in a few weeks? https://github.com/mozilla-mobile/reference-browser/

Reply:
Good thinking, done.